yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -17,20 +17,46 @@
     {% comment %} HACK: For some reason this allows formatting to work, do not remove. {% endcomment %}
     <div class="filler"></div>
     <div class="border">
-        {% if edit == True %}
-        <div class="edit_button">
+     <div class=edit_button>
+        {% if edit == False %}
             <form method="get" action="{% url 'edit_page' workflow.id version.id %}">
                 {% csrf_token %}
-                <button class="button" type="submit"> Edit Workflow</button>
+                <button type="submit"> Edit Workflow</button>
             </form>
-            <div class="delete">
-                <button class="button" onclick="deleteFunctionWorkflows()">Delete Workflow</button>
+            <div class=delete>
+                <button onclick="deleteFunc()">Delete Workflow</button>
             </div>
             <script>
-                function deleteFunctionWorkflows() {
-                    if (confirm("Do you really want to delete this workflow?")) {
-                        document.location.href="{% url 'delete_workflows' workflow.id %}";
-                    }
+                function deleteFunc() {
```

Comment:
I think you should call this deleteWorkflow(), it's obvious it's a function but without reading the code body it is unclear what is getting deleted.